### DISCUSSION TITLE
Optparse over Argparse

### DISCUSSION THREAD

**Discussion by alaurie** [almenscorner/IntuneCD#47] [General]
Hi,

I was wondering if there was a particular reason to use Optparse over Argparse as Optparse is deprecated these days.

**Comment by almenscorner** [comment]
Hi Alex,

No particular reason, in an upcoming version I have changed `optparse` to `argparse` :)

**Comment by almenscorner** [comment]
This has now been changed as of 1.1.2 🎉  #50
